src/renderer/Config.tsx

```diff
@@ -0,0 +1,62 @@
+import {BT, HTF, IF, TF, VTF, ZF} from "./const";
+
+export class SceneSettings {
+  timingFunction = TF.constant;
+  timingConstant = "1000";
+  imageTypeFilter = IF.any;
+  zoomType = ZF.none;
+  effectLevel = 5;
+  horizTransType = HTF.none;
+  vertTransType = VTF.none;
+  crossFade = false;
+  backgroundType = BT.blur;
+  backgroundColor = "#000000";
+  playFullGif = false;
+  overlaySceneID = 0;
+  overlaySceneOpacity = 0.5;
+  textKind = "";
+  textSource = "";
+
+  blinkColor = "#FFFFFF";
+  blinkFontSize = 20;
+  blinkFontFamily = "Arial Black,Arial Bold,Gadget,sans-serif";
+
+  captionColor = "#FFFFFF";
+  captionFontSize = 8;
+  captionFontFamily = "Helvetica Neue,Helvetica,Arial,sans-serif";
+
+  captionBigColor = "#FFFFFF";
+  captionBigFontSize = 12;
+  captionBigFontFamily = "Arial Black,Arial Bold,Gadget,sans-serif";
+}
+
+export class RemoteSettings {
+  tumblrDefault = "BaQquvlxQeRhKRyViknF98vseIdcBEyDrzJBpHxvAiMPHCKR2l";
+  tumblrOverlay = "G4iZd6FBiyDxHVUpNqtOTDu4woWzfp8WuH3tTrT3MC16GTmNzq";
+  tumblrOther = ["BaQquvlxQeRhKRyViknF98vseIdcBEyDrzJBpHxvAiMPHCKR2l",
+    "G4iZd6FBiyDxHVUpNqtOTDu4woWzfp8WuH3tTrT3MC16GTmNzq",
+    "UHpRFx16HFIRgQjtjJKgfVIcwIeb71BYwOQXTMtiCvdSEPjV7N",
+    "JFNLu3CbINQjRdUvZibXW9VpSEVYYtiPJ86o8YmvgLZIoKyuNX"];
```

Comment:
Can we at least keep the `tumblrDefault` and `tumblrOverlay` keys? It works a lot better using separate key for those two.

Reply:
Why? Were you never able to keep a single queue of Tumblr requests?

Reply:
Why? Were you never able to keep a single queue of Tumblr requests?

Reply:
No, I did not implement a single queue as of yet. Currently, each `HeadlessScenePlayer` instances runs the following algorithm on mount:
```
Loop over each source once
  Generate a promise for the source -- This promise is either a local directory, the cache for a source, or the first page
  Run the promise
    Record the URLs
    If this promise is for a remote source
      Queue up the next page in a promise queue
While promiseQueue is not empty
  Run promise
    If we got results
      Record the URLs (in the correct list)
      Queue up the next page
```
I guess this could be delegated to a single queue.. but without 2 API keys, processing will be twice as slow. (1 per 16 seconds)

Reply:
No, I did not implement a single queue as of yet. Currently, each `HeadlessScenePlayer` instances runs the following algorithm on mount:
```
Loop over each source once
  Generate a promise for the source -- This promise is either a local directory, the cache for a source, or the first page
  Run the promise
    Record the URLs
    If this promise is for a remote source
      Queue up the next page in a promise queue
While promiseQueue is not empty
  Run promise
    If we got results
      Record the URLs (in the correct list)
      Queue up the next page
```
I guess this could be delegated to a single queue.. but without 2 API keys, processing will be twice as slow. (1 per 16 seconds)

Reply:
Went ahead and removed `tumblrOther`.